spec/support/coronavirus_landing_page_steps.rb

```diff
@@ -124,6 +141,16 @@ def then_i_can_see_the_timeline
     expect(page).to have_selector(".covid-timeline__item .gem-c-govspeak", text: "Face coverings are mandatory in shops")
   end
 
+  def then_i_can_see_the_timeline_for_england
+    expect(page).to have_selector(".js-covid-timeline .govuk-heading-m", text: "Changes in England")
+    expect(page).to have_selector(".js-covid-timeline .govuk-heading-m", text: "Changes in Wales", visible: false)
```

Comment:
Do these need to be selector specific? 

I imagine it'd be a cleaner read with `expect(page).to have_content` and `expect(page).to_not have_content` assertions

Reply:
Strangely enough that doesn't work. I think it's because the content is in the page source, but is only visually hidden.

Reply:
I see. `have_content` does consider visibility (you can actually pass in a [first argument of `:visible` or `:all`](https://www.rubydoc.info/gems/capybara/Capybara/Node/Matchers#assert_text-instance_method))

I imagine we've got a situation where `have_content` works for the Javascript version, as that actually involves a page being rendered. However your failing situation occurs without JS and the page is accessed through rack-test, in that scenario I'd imagine we have the issue where `expect(page).to have_selector(".js-covid-timeline .govuk-heading-m", text: "Changes in Wales", visible: false)` ignores the visible argument and ends up being a bit of a bogus test.